DESCRIPTION

```diff
@@ -26,18 +40,22 @@ LazyData: true
 URL: https://r-lib.github.io/webfakes,
     https://github.com/r-lib/webfakes#readme
 BugReports: https://github.com/r-lib/webfakes/issues
-RoxygenNote: 7.1.1
+RoxygenNote: 7.1.1.9000
 Config/testthat/edition: 3
 Roxygen: list(markdown = TRUE)
 Imports:
     stats,
     tools,
     utils
 Suggests:
+    knitr,
     callr,
     curl,
     glue,
     jsonlite,
     testthat (>= 3.0.0),
-    withr
+    withr,
+    xml2,
+    rmarkdown
 Encoding: UTF-8
+VignetteBuilder: knitr
```

Comment:
and this is probably because of my creating the vignette via usethis. :thinking:

Reply:
vignettes need xml2 now?